jolly-jellyfish/src/page_maker/templates/page_maker/webpage_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div>
```

Comment:
```suggestion
{% block content %}
<h2>Delete Page</h2>
<div>
```
As above, styling concerns.